app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,95 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+/**
+ * 상품(Product)과 관련된 비즈니스 로직을 담당합니다.
```

Comment:
`Product`와 주석 사이에 커플링이 있습니다. 다음과 같이 수정하는 것으로도 충분할 것 같아요.
```suggestion
 * 상품과 관련된 비즈니스 로직을 담당합니다.
```